src/traffic_layout/engine.cc

```diff
@@ -566,6 +332,99 @@ RunrootEngine::copy_runroot(const std::string &original_root, const std::string
   } else {
     path_map[LAYOUT_PREFIX] = ".";
   }
+
+  YAML::Emitter yamlfile;
+  // emit key value pairs to the yaml file
+  yamlfile << YAML::BeginMap;
+  for (uint i = 0; i < dir_vector.size(); i++) {
+    yamlfile << YAML::Key << dir_vector[i];
+    yamlfile << YAML::Value << path_map[dir_vector[i]];
+  }
+  yamlfile << YAML::EndMap;
+
+  // create the file
+  std::ofstream f(Layout::relative_to(ts_runroot, "runroot.yaml"));
+  f << yamlfile.c_str();
+}
+
+// check if user want to force remove the ts_runroot
+// return true if user replies Y
+static bool
+check_remove_force()
+{
+  // check for Y/N 3 times
```

Comment:
I don't think we need to ask for Y 3 times.  Adding --force and asking once is good enough.

Reply:
It's once. The comment is a bit confusing. If the user provides an invalid input it will continue to ask.